docker-compose.yml

```diff
@@ -665,3 +665,30 @@ services:
     labels:
       - "logging=true"
     logging: *default-logging
+  lmb-ldes-consumer:
+    image: redpencil/ldes-consumer:0.8.0-rc1
+    depends_on:
+      - database
+    environment:
+      LDES_STREAM: "https://mandatenbeheer.lblod.info/streams/ldes/public"
+      LDES_ENDPOINT_VIEW: "https://mandatenbeheer.lblod.info/streams/ldes/public/1"
+      CRON_PATTERN: "*/1 * * * *" # run the job every 1 minute
+      REPLACE_VERSIONS: true 
+      MU_SPARQL_ENDPOINT: 'http://database:8890/sparql'
+      MU_APPLICATION_GRAPH: "http://mu.semte.ch/graphs/mandaten-staging"
+      PERSIST_STATE: "true"
+    labels:
+        - "logging=true"
+    restart: always
+    logging: *default-logging
+  lmb-upsert-ldes-entity:
+    image: lblod/ldes-make-object-service:latest
+    depends_on:
+      - lmb-ldes-consumer
+    environment:
+      LANDING_ZONE_GRAPH: "http://mu.semte.ch/graphs/mandaten-staging"
+      TARGET_GRAPH: "http://mu.semte.ch/graphs/mandaten"
```

Comment:
can this be renamed, to e.g. `http://mu.semte.ch/graphs/lmb/mandaten`? Cause we have many types of mandaten in  loket